src/components/PrimaryNav.js

```diff
@@ -54,7 +54,11 @@ const PrimaryNav = forwardRef(
         </Flex>
 
         <Box display={['block', 'block', 'none']} mr={{ sm: '5' }}>
-          <button onClick={handleToggle}>
+          <button
+            onClick={handleToggle}
+            aria-expanded={isVisible}
```

Comment:
@magnificode should this be set to true on desktop?

Reply:
@M0nica nope! Because the entire activator for the menu is hidden, this will be skipped over altogether by screen readers on desktop.

Reply:
Do you know how I can set a`aia-expanded` to`false` on desktop since `isVisible` is true on desktop?